scripts/functions.sh

```diff
@@ -646,77 +668,80 @@ function check_args_and_apply() {
         exit 1
       fi
       ;;
-    --enable_filter) # Enable flow filter
-      if [[ "$3" == "flows" || "$3" == "metrics" ]]; then
-        if [[ "$value" == "true" || "$value" == "false" ]]; then
-          edit_manifest "filter_enable" "$value" "$2"
-        else
-          echo "invalid value for --enable_filter"
-        fi
+    *enable_all) # Enable all features
+      defaultValue "true"
+      if [[ "$value" == "true" || "$value" == "false" ]]; then
+        edit_manifest "pktdrop_enable" "$value" "$2"
+        edit_manifest "dns_enable" "$value" "$2"
+        edit_manifest "rtt_enable" "$value" "$2"
+        edit_manifest "network_events_enable" "$value" "$2"
```

Comment:
we are missing a couple of features here :) pkt_xlat and udnMapping if u can add while u are here that will be good else I can do it post merge

Reply:
sure that should not be a huge change anyways